mozfldp/random_data_gen.py

```diff
@@ -43,82 +54,85 @@ def generate_random_data(g_prms):
     return _gen_data_and_add_user_data(_gen_random_data, g_prms)
 
 
-def transform_data_for_simulator_format(df, g_prms):
+def transform_data_for_simulator_format(df, g_prms, verbose=False):
     """
     Transforms a Pandas DataFrame returned by the generation functions into a format that is usable by the simulator.
     See notebook for the specific format.
     """
     labels = []
     feats = []
     for i in range(g_prms.num_users):
-        client_df = df[df.user_id == i].drop(df.columns[0], axis=1)
-        labels.append(list(client_df.labels))
-        client_feats = client_df.drop(columns=["user_id", "labels"]).to_records(
+        client_df = df[df.user_id == i]  # .drop(df.columns[0], axis=1)
+        labels.append(list(client_df.label))
+        client_feats = client_df.drop(columns=["user_id", "label"]).to_records(
             index=False
         )
-        
+
         # convert tuples to lists
         client_feats = [list(feat) for feat in client_feats]
-        
+
         feats.append(client_feats)
-        print("Loading data for client", i)
```

Comment:
I don't think we even need this print. It's doesn't really add that much value to debugging I think. Should we just remove `verbose` altogether?

Reply:
Good point - let's just drop this.